src/ParameterType.php

```diff
@@ -2,8 +2,13 @@
 
 namespace Doctrine\DBAL;
 
+use Doctrine\DBAL\Types\Type;
+
 /**
  * Contains statement parameter types.
+ *
+ * @phpstan-type ParameterTypeOfValue self::*|Type|string|null
```

Comment:
You can remove this one I think, PHPStan understands `psalm-type`, right?

Reply:
Yes, you are right.